docs/workflows/data/google-ml.md

```diff
@@ -0,0 +1,28 @@
+# GCP ML
+
+The GCP (Google Cloud Platform) ML node allows a workflow to send data to Google ML and store the resulting predictions on the workflow payload.
```

Comment:
Like with BigQuery, add a "Google ML" link to one of their pages explaining what the service is.